### ISSUE TITLE
hexadecimal number recognition in lexer

### ISSUE THREAD

**Issue by MDorwig** [fbb-git/icmake#2]
the pattern for hexadecimal numbers should be 0x[0-9a-fA-F]+

**Comment by fbb-git** [comment]
Dear MDorwig, you wrote:

> the pattern for hexadecimal numbers should be 0x[0-9a-fA-F]+

Thanks! No idea why this pattern wasn't used in the lexer file Lost in the
mists of time, I guess. It'll be fixed at the next upgrade.

## 

```
Frank B. Brokken
Center for Information Technology, University of Groningen
(+31) 50 363 9281 
Public PGP key: http://pgp.surfnet.nl
Key Fingerprint: DF32 13DE B156 7732 E65E  3B4D 7DB2 A8BE EAE4 D8AA
```

**Comment by fbb-git** [comment]
Hi Martin,

AFAICS the hex-number recognition has been fixed in 8.00.00. Thx!
